webpack/assets/javascripts/react_app/components/SettingsTable/components/SettingValueEdit.js

```diff
@@ -59,8 +59,9 @@ const SettingValueEdit = ({ setting, updateSetting }) => {
     setLoading(false);
   };
 
-  const successCallback = submitValue => {
-    updateSetting(submitValue);
+  const successCallback = (response, submitValue) => {
+    const responseValue = response?.data?.value;
+    updateSetting(responseValue || submitValue);
```

Comment:
What would it mean if the responseValue is empty?

Reply:
:thinking: I guess if the callback is successful, then we never expect an empty value.